File: prak_lat1 php/Disk.php
<?php

class Disk{

	// Attributes
	private $type;
	private $capacity;
	private $price;

	// Constructor
	public function __construct($type, $capacity, $price){
		$this->type = $type;
		$this->capacity = $capacity;
		$this->price = $price;
	}

	// Set & Get Methods
	public function setType($type){
		$this->type = $type;
	}

	public function setCapacity($capacity){
		$this->capacity = $capacity;
	}

	public function setPrice($price){
		$this->price = $price;
	}

	public function getType(){
		return $this->type;
	}

	public function getCapacity(){
		return $this->capacity;
	}

	public function getPrice(){
		return $this->price;
	}

}

?>

File: prak_lat1 python/Disk.py
class Disk:
	# Constructor
	def __init__(self, dtype, capacity, price):
		self.dtype = dtype
		self.capacity = capacity
		self.price = price

	# Set & Get methods
	def set_type(self, dtype):
		self.dtype = dtype

	def set_capacity(self, capacity):
		self.capacity = capacity

	def set_price(self, price):
		self.price = price

	def get_type(self):
		return self.dtype

	def get_capacity(self):
		return self.capacity

	def get_price(self):
		return self.price